# Satellite Position
float32 position_x
float32 position_y
float32 position_z

# Satellite Orientation
float32 orientation_x
float32 orientation_y
float32 orientation_z
float32 orientation_w

# Satellite Velocity
float32 velocity_x
float32 velocity_y
float32 velocity_z

# Current time
int32 time